distbench_test_sequencer_test.cc

```diff
@@ -151,10 +151,7 @@ TEST(DistBenchTestSequencer, NonEmptyGroup) {
   ASSERT_EQ(s2_1_echo->second.successful_rpc_samples_size(), 10);
 }
 
-void RunIntenseTraffic(const char* protocol) {
-  DistBenchTester tester;
-  ASSERT_OK(tester.Initialize(6));
-
+TestSequence GetPingIntense(const char* protocol) {
```

Comment:
GetPingIntense is not the best name IMO. Maybe something like IntenseTrafficTestSequence?

Reply:
Renamed as suggested.